liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/CreateServerMojo.java

```diff
@@ -67,10 +105,75 @@ protected void doExecute() throws Exception {
             serverTask.execute();
             // copy files _after_ we create the server
             copyConfigFiles(true);
+            exportParametersToXml();
             log.info(MessageFormat.format(messages.getString("info.server.create.created"), serverName, serverDirectory.getCanonicalPath()));
         } else {
             // server exists - copy files over
             copyConfigFiles();
+            exportParametersToXml();
```

Comment:
with overwrite going away, could you remove the else and just have the following 2 lines after the if?
         copyConfigFiles(true);
         exportParametersToXml();